core/src/test/scala/integration/kafka/api/TransactionsBounceTest.scala

```diff
@@ -76,12 +75,11 @@ class TransactionsBounceTest extends IntegrationTestHarness {
 
   override protected def brokerCount: Int = 4
 
-  @nowarn("cat=deprecation")
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
   @MethodSource(Array("getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17961"))
   def testWithGroupId(quorum: String, groupProtocol: String): Unit = {
     testBrokerFailure((producer, groupId, consumer) =>
-      producer.sendOffsetsToTransaction(TestUtils.consumerPositions(consumer).asJava, groupId))
+      producer.sendOffsetsToTransaction(TestUtils.consumerPositions(consumer).asJava, consumer.groupMetadata()))
```

Comment:
Please remove this duplicate test case :)

Reply:
> Please remove this duplicate test case :)

Okay.